app/controllers/schools/participants_controller.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+class Schools::ParticipantsController < Schools::BaseController
+  skip_after_action :verify_authorized
+  skip_after_action :verify_policy_scoped
```

Comment:
We definitely shouldn't remove `verify_authorized` from the base controller and I definitely need to add authentication to this so that it doesn't need to be skipped 😬 

I'm not sure about the other one, though

Reply:
I agree, every time I see that `skip_after_action` I think that having them in base controller is a mistake :P